hr_holidays_public_generator/wizard/hr_holidays_public_generator_view.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
```

Comment:
@yweng8111  IMHO Replace to `<odoo>`

Reply:
Thanks for the review. I will fix it now.